uwb/example/lib/host_widgets/arrow.dart

```diff
@@ -0,0 +1,36 @@
+import 'dart:math' as math;
+
+import 'package:flutter/material.dart';
+
+/// The [Arrow] class that generates an arrow to point at the location of the connected device.
+class Arrow extends StatelessWidget {
+  /// The [Arrow] constructor.
+  const Arrow({Key? key, required this.x, required this.y, required this.angle})
+      : super(key: key);
+
+  /// Variable for x-axis of the arrow.
+  final double x;
+
+  /// Variable for y-axis of the arrow.
+  final double y;
+
+  /// Variable for arrow direction.
```

Comment:
Better description would be:
```suggestion
  /// The direction in which the arrow is pointing.
```